spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/ModuleCommands.java

```diff
@@ -61,14 +62,18 @@
 
 	private static final String REGISTER_MODULE = "module register";
 
+	private static final String IMPORT_APPS = "apps import";
```

Comment:
Right. I considered `app all` which would be similar to `stream all` but that didn't seem very intuitive. Maybe we should instead consider the other word in the command... I guess `app import` is okay though (could imply 1 *or more*), and maybe we just need to make the existing `app register` (once `module register` is renamed) accept 1 *or more* itself.